Doc/reference/datamodel.rst

```diff
@@ -1607,10 +1607,11 @@ Basic customization
    class).  The return value of :meth:`__new__` should be the new object instance
    (usually an instance of *cls*).
 
-   Typical implementations create a new instance of the class by invoking the
+   Typical custom implementations create a new instance of the class by invoking the
```

Comment:
Disagree with this addition, it does not clarify or improve what is said.

The line is about typical implementations of a `__new__` method, meaning any code written in stdlib or by other developers. «custom» does not make sense here; we are not talking about «custom classes» (meaning non-builtin), but literally about writing the code for the method in your class.

Reply:
I think I was pointing out this sentence is about implementations in other classes, not in _object_ itself. But maybe it doesn’t need to be explicit, especially if the next sentence about the base implementation stays (or if specification of the concrete _object_ type is moved to e.g. library/stdtypes.rst). Agree I picked a confusing word.

Reply:
Oh, this whole section is about writing your `__new__` method in your python class, not `object` itself! (line 1599 is confusing)

Reply:
```suggestion
   Typical implementations create a new instance of the class by invoking the
```